lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/MavenProjectCache.java

```diff
@@ -68,6 +76,7 @@
 
 	private final List<Consumer<MavenProject>> projectParsedListeners = new ArrayList<>();
 	private Properties systemProperties;
+	private URIResolverExtensionManager resolverExtensionManager = new URIResolverExtensionManager();
```

Comment:
Don't instainciate resolverExtensionManager, but use the resolverExtensionManager  from MavenLemminxExtension getted in start method.

Reply:
ok